autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -369,5 +395,6 @@ def handle(self, *args, **options):
 
         instructors, sections, meetings, courses = get_course_data(depts_terms)
         save_models(instructors, sections, meetings, courses, term, terms, options)
+        save_terms(term, terms, options)
```

Comment:
Not really necessary in this PR but I would like if we could refactor this file to just use `terms`, I don't see why we couldn't set `terms = [options['term']]` instead of making a single term a special case

Reply:
Yeah I agree - it's super obvious in hindsight idk why I didn't do it before